simple/csharp/rename-mehod_after.cs

```diff
@@ -0,0 +1,12 @@
+class Customer 
+{
+  String name = "name";
+  String surname = "surname";
```

Comment:
Rename surname to lastName (project uses american english). Do the same change in all other versions of the example.